dandi/cli/cmd_download.py

```diff
@@ -4,6 +4,29 @@
 from .base import devel_option, map_to_click_exceptions
 
 
+class ChoiceList(click.ParamType):
+    name = "choice-list"
+
+    def __init__(self, values):
+        self.values = set(values)
+
+    def convert(self, value, param, ctx):
+        if value is None or isinstance(value, set):
```

Comment:
I think it would be wortwhile to reincarnate unit testing of `cli` interface, while mocking out actual functionality (`dandi.download.download` in this case)

Reply:
I added unit tests for the `--download` option.  Is that what you had in mind?

Reply:
YES!  Thank you!